everett/manager.py

```diff
@@ -71,6 +101,30 @@ def get_parser(parser):
     return parser
 
 
+def get_key_from_envs(envs, key, namespace=None):
+    """Return the value of a key from the given dict respecting namespaces.
+
+    Data can also be a list of data dicts.
+    """
+    # if it barks like a dict, make it a list
+    # have to use `get` since dicts and lists
+    # both have __getitem__
```

Comment:
Yeah. Could do that. I've so far not come up with a better way to detecting an iterable vs. a dict-like thing. I can't use `isinstance(envs, dict)` because `os.environ` isn't a dict, and I can't just use `isinstance(envs, (list, tuple))` because `reverse(list)` returns a generator... If you know of a better and more clear way to do this I'd be very happy to know about it.